editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
To be honest, I don't have any preference. The advantage of `reselect` is that it's kind of the default with Redux, more developers already know how to use it. But at the same time, `rememo` is probably easier to learn.